libsoftwarecontainer/src/gateway/iptableentry.h

```diff
@@ -67,6 +67,18 @@ class IPTableEntry
      */
     ReturnCode applyRules();
 
+    /**
+     * @brief Interprets a rule to iptables applicable string
+     * @return string indicating interpreted rule
+     */
+    std::string interpretRule(Rule rule);
+
+    /**
+     * @brief Interprets defaultTarget rule to iptables applicable string
```

Comment:
Kursat - that explanation makes for a better comment then I think, put the explanation there.